src/app/contact/contact.component.html

```diff
@@ -0,0 +1,339 @@
+<!-- Start ignoring BootLintBear-->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+<link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
+<!-- Stop ignoring BootLintBear-->
+<nav class="navbar navbar-default nav-about">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand about-navbar" href="//susper.com">
+        <img alt="brand" class="navbar-logo" src="../../assets/images/susper.svg">
+      </a>
+    </div>
+    <p class="navbar-text navbar-right">Contact Us</p>
+  </div>
+</nav>
+
+<div class="image-banner">
+  <img src="../../assets/images/boat.png" class="img-responsive banner">
+</div>
+
+<div class="container">
+  <br>
+  <div class="about-sub-details">
+    <div class="row contact-title">
+      <h2 class="bold">Contact Us</h2>
+      <br>
+    </div>
+    <br>
+    <div class="row">
+      <div class="col-lg-12 col-sm-12 sub-details">
+        <h5 class="bold">Susper</h5>
+        <p>93 Mau Than Street<br> Can Tho<br> Viet Nam<br> Phone +84 (0) 907 65 29 27<br> Email: support@susper.net<br> Board
+          of Directors: Phuc Hau Dang<br> Susper Ltd. is registered in Can Tho, Viet Nam.</p>
+      </div>
+    </div>
+    <!--Contact Us details ends-->
+  </div>
+  <!--Contact-sub-details ends here-->
+
+  <br>
+
+  <div class="contact-sub-details">
+    <div class="row">
+      <div class="col-lg-12 col-sm-12 contact-sub-details">
+        <p>Report a safety or abuse issue affecting our products.<br> If you know of a safety or abuse problem with any of Susper's
+          services, we'd like to hear about it right away. Please use our <a (click)="open()">contact</a> form to report the
+          issue.
+        </p>
+      </div>
+
+    </div>
+  </div>
+  <!--contact-sub-details ends here-->
+
+</div>
+<!--container ends here-->
+<footer>
+  <!--footer navigation bar goes here-->
+  <app-footer-navbar></app-footer-navbar>
+</footer>
+
+<modal #myModal>
+  <div>
+    <a (click)="close()" class="close-btn"> <i class="fa fa-times" aria-hidden="true" style="font-size: 2em"></i> </a>
+ 
+
+  </div>
+  <form class="form-horizontal" method="POST" action="http://formspree.io/office@fossasia.org">
```

Comment:
it is a platform that use to send emails without any backend codes. if you can check loklak.net contact page you can see the same code there.